en/docs/administer/logging-and-monitoring/monitoring/monitoring-tcp-based-messages.md

```diff
@@ -64,33 +63,31 @@ To download TCPMon from Apache and run the tool:
             ./tcpmon.sh
     ```
 
+## Other Usages of TCPMon
 
+TCPMon is primarily used for message monitoring. Additionally, TCPMon can also be used for sending requests to web services and as a proxy service.
 
-# Other Usages of TCPMon
-
-TCPMon is primarily used for message monitoring. Additionally, TCPMon can also be used for sending requests to web services and as a proxy service. Refer [Starting TCPMon](https://docs.wso2.com/display/Carbon440/Starting+TCPMon) for details on how to start the tool.
-
--   [Sending Requests for Web Services](#OtherUsagesofTCPMon-SendingRequestsforWebServices)
--   [As a Proxy](#OtherUsagesofTCPMon-AsaProxy)
--   [Advanced Settings](#OtherUsagesofTCPMon-AdvancedSettings)
+-   Sending Requests for Web Services
+-   As a Proxy
+-   Advanced Settings
 
 #### Sending Requests for Web Services
 
 TCPMon can also be used as a request sender for Web services. The request SOAP message can be pasted on the send screen and sent directly to the server.
 
-![]({{base_path}}/assets/attachments/45946410/46206514.png)
+<a href="{{base_path}}/assets/img/administer/tcpmon_screen4.png"><img src="{{base_path}}/assets/img/administer/tcpmon_screen4.png" alt="TCPMon sending requests" name="TCPMon sending requests" width="520"></a>
```

Comment:
This screen including the others are outdated.

Reply:
Yes. For now let's merge and see what needs to be done with this page.